src/main/java/CashDrawerController.java

```diff
@@ -0,0 +1,55 @@
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+
+@RestController
```

Comment:
Can I ask why we need a web controller here? This is actually an example of trying to get too fancy without the fundamental requirements of the app being met.

I don't think we need Spring boot at all.

Reply:
Same. I love the ambition, but lets walk before we start doing somersaulting backflips first. 
The specifications of this app really dont ask for a web app and speaking as someone that reviews this homework, fancy features like SpringBoot and web-apps dont impress me if the basic requirements of the ask itself arent actually met.